brave/ui/index.js

```diff
@@ -0,0 +1,8 @@
+const launchMetamaskUi = require('../../ui')
+const actions = require('./app/store/actions')
+
+module.exports = (opts, cb) => {
+  launchMetamaskUi(opts, cb)
+  const { backgroundConnection } = opts
+  actions.setBackgroundConnection(backgroundConnection)
```

Comment:
I ended up having to do it this way as the `background` var is not exportable in the parent actions file, and we need that one set as well for the original actions. This way `background` is available in both the parent and extended context. Also, something like this does not work:

```
var localBackground = null
function newSetBackground (conn) {
  MetaMaskActions._setBackgroundConnection(conn)
  localBackground = conn
}

MetaMaskActions._setBackgroundConnection = newSetBackground
```

This will produce a maximum call stack size exception, as once we override it we end up just calling our new function over and over.